### PR TITLE
New Dungeons and Dragons 5th Edition Proficiency Cheat Sheet

### PR DESCRIPTION
**Pull request by doorknob22**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x ] New Instant Answer
    - [x ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**

###### Description of changes

A cheat sheet for Dungeons and Dragons 5th edition of what a character's proficiency bonus is based on their level.

###### Which issues (if any) does this fix?

N/A

###### People to notify (@mention interested parties)
@tagawa 

---

Instant Answer Page: https://duck.co/ia/view/dnd5e_proficiency_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@doorknob22 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/dnd5e-proficiency.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "id": "dnd5e_proficiency_cheat_sheet",
+    "name": "dnd5e_proficiency",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/dnd5e-proficiency.json]: This is what appears at the top of the Instant Answer so it should be a human-friendly name.

**Comment by tagawa** [comment]
@doorknob22 Thanks very much for sending the pull request - it seems to be working fine! It's now installed on the beta server here: https://beta.duckduckgo.com/?q=dnd5e+proficiency+cheat+sheet&iax=1

As I mentioned, it's a bit short for a cheat sheet so the next step is to consider how to add other D&D terms, rules, etc., so that more people will find it helpful.

**Pull request closed without merging**